build.assets/robotest_run_nightly.sh

```diff
@@ -7,7 +7,8 @@ DOCKER_STORAGE_DRIVERS="overlay2"
 
 declare -A UPGRADE_MAP
 # gravity version -> list of OS releases to test upgrades on
-UPGRADE_MAP[5.5.0]="centos:7 ubuntu:16"
+UPGRADE_MAP[6.0.9]="centos:7 ubuntu:16"
```

Comment:
Is this not working due to version skew policy?

Reply:
Yeah. I think it makes sense to test only officially supported upgrade paths.